### PR TITLE
change store app SP name to devportal

### PR DESCRIPTION
**Pull request by isharac**
atm store app SP name is 'admin_store'
which is changed to 'admin_devportal'

### CODE REVIEW

**Comment by bhathiya** [comment]
shall we rename all apps as `apim_publisher` instead of `admin_publisher`?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/services/login/idp.jag [hunk 1]
```diff
@@ -55,16 +55,24 @@
     var SystemApplicationDAO = Packages.org.wso2.carbon.apimgt.impl.dao.SystemApplicationDAO;
     var systemApplicationDAO = new SystemApplicationDAO();
     var clientId ;
-    systemApplicationDTO = systemApplicationDAO.getClientCredentialsForApplication(STORE_CLIENT_APP_NAME);
+    // this is to support migration from admin_store to admin_devportal
+    systemApplicationDTO = systemApplicationDAO.getClientCredentialsForApplication(STORE_CLIENT_APP_NAME_OLD);
```
Comment (bhathiya) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/services/login/idp.jag]: can't we rename the app here?

Comment (isharac) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/services/login/idp.jag]: it includes app name update in AM_SYSTEM_APPS table and rename in the identity server side as well.
Do we need to do it?

**Comment by tmkasun** [comment]
> shall we rename all apps as `apim_publisher` instead of `admin_publisher`?

Yes agree with this change

**Comment by isharac** [comment]
> > shall we rename all apps as `apim_publisher` instead of `admin_publisher`?
> 
> Yes agree with this change

+1

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/services/login/idp.jag (hunk 1 above)
Comment (isharac) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/services/login/idp.jag]: As per the offline chat we decided to keep the migrated apps as it is.
Only new apps will be created as 'apim_devportal' and 'apim_publisher'

**Pull request merged**